src/api/endpoints/auth/AuthEndpoint.ts

```diff
@@ -28,8 +28,8 @@ export default class AuthEndpoint extends ApiEndpoint {
    *
    * @virtual
```

Comment:
do we know why this method is marked as virtual? It shouldn't ever change should it?

Reply:
I think just some it's just some copy pasta never caught